dataprofiler/tests/profilers/test_categorical_options.py

```diff
@@ -0,0 +1,25 @@
+from dataprofiler.profilers.profiler_options import CategoricalOptions
+from dataprofiler.tests.profilers.test_base_column_options import TestBaseColumnOptions
+
+
+class TestCategoricalOptions(TestBaseColumnOptions):
+    
+    option_class = CategoricalOptions
+    
+    def test_init(self, *mocks):
+        super().test_init(*mocks)
+    
+    def test_set_helper(self, *mocks):
+        super().test_set_helper(*mocks)
+
+    def test_set(self, *mocks):
+        super().test_set(*mocks)
+    
+    def test_validate_helper(self, *mocks):
+        super().test_validate_helper(*mocks)
+    
+    def test_validate(self, *mocks):
+        super().test_validate(*mocks)
+
+    def test_is_prop_enabled(self, *mocks):
+        super().test_is_prop_enabled(*mocks)
```

Comment:
I'm not super familiar with mock but if you don't have any on this class why are you passing *mocks through?

Reply:
This comment goes for all the other test files too, you aren't mocking anything out in any of them so is this kwarg still necessary?

Reply:
@JGSweets would know more, I'm not super familiar

Reply:
You know, that is a good catch. I missed that in the previous PRs too I believe. There would be no need for the `*mocks` if no mock decorator was applied (it may be required if it were applied to the base class).

Reply:
Makes sense, it was included in the original test_profiler_options since mocks were being used in those files, but there hasn't been a need for them in these tests - I will go ahead and delete them from all files.